ndex2/client.py

```diff
@@ -950,6 +1327,26 @@ def update_network_user_permission(self, userid, networkid, permission):
     def grant_networks_to_group(self, groupid, networkids, permission="READ"):
         """
         Set group permission for a set of networks
+	
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		#find the group id by the username
+		user = "INSERT YOUR USERNAME HERE"
+		group_ids = [g.gr_gid for g in grp.getgrall() if user in g.gr_mem]
+		gid = pwd.getpwnam(user).pw_gid
+		group_ids.append(grp.getgrgid(gid).gr_gid)
+		print (group_ids)
+
+		group_id = group_ids[0]
+		network_ids = 'ENTER THE NETWORK ID HERE'
+		#permission can be changed to edit
+		my_ndex.grant_networks_to_group(group_id, network_ids, permission=READ)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client